yew-router-macro/Cargo.toml

```diff
@@ -13,8 +13,8 @@ proc-macro = true
 [dependencies]
 syn = "1.0.2"
 quote = "1.0.1"
-yew-router-route-parser = { version = "0.13.0", path = "../yew-router-route-parser"}
+yew-router-route-parser = { version = "0.14.0", path = "../yew-router-route-parser"}
 proc-macro2 = "1.0.1"
 
 [dev-dependencies]
-yew-router = { version = "0.13.0", path = "../yew-router" } # This should probably be removed, it makes the deploy process much more annoying.
\ No newline at end of file
+yew-router = { version = "0.14.0", path = "../yew-router" } # This should probably be removed, it makes the deploy process much more annoying.
```

Comment:
I haven't run into this issue anymore when publishing. Have you @hgzimmerman?

Reply:
My problem with this "reverse dev dependency" is that yew router macro needs to be released first, but it depends on the _next_ (soon to be released) version of yew router, so I need to comment it out before I publish it. It adds a decent amount of friction and I tend to forget to re-uncomment it about half the time. 

Removing it would mean that a doc-test wouldn't work (but I guess it could be moved elsewhere?).

Reply:
Yeah, I used to run into this exact problem with `yew-macro` having a dev dep on `yew` but for some reason it doesn't complain anymore so I thought it was resolved in a recent update to `cargo`

Reply:
Interesting, I just do this comment-uncomment dance out of habit at this point. I'd be thrilled to discover cargo is smart enough to avoid this now.